misk-hibernate/src/test/kotlin/misk/hibernate/ReflectionQueryFactoryTest.kt

```diff
@@ -420,6 +420,43 @@ class ReflectionQueryFactoryTest {
     }
   }
 
+  @Test
+  fun deleteFailsWithOrderBy() {
+    transacter.transaction { session ->
```

Comment:
/nit is it worth even setting this data up?

Reply:
I changed the test to make sure the delete didn't go through